datahub-web-react/src/app/entity/shared/tabs/Documentation/components/DescriptionEditor.tsx

```diff
@@ -10,15 +10,22 @@ import TabToolbar from '../../../components/styled/TabToolbar';
 import { GenericEntityUpdate } from '../../../types';
 import { useEntityData, useEntityUpdate, useRefetch } from '../../../EntityContext';
 import { useUpdateDescriptionMutation } from '../../../../../../graphql/mutations.generated';
+import { DiscardDescriptionModal } from './DiscardDescriptionModal';
 
 export const DescriptionEditor = ({ onComplete }: { onComplete?: () => void }) => {
     const { urn, entityType, entityData } = useEntityData();
     const refetch = useRefetch();
     const updateEntity = useEntityUpdate<GenericEntityUpdate>();
     const [updateDescriptionMutation] = useUpdateDescriptionMutation();
 
-    const description = entityData?.editableProperties?.description || entityData?.properties?.description || '';
+    const localStorageDictionary = localStorage.getItem('editedDescriptions');
```

Comment:
Can we store the constant "editedDescription" is a separate constants file? Maybe we can put it inside `utils.ts`?

Reply:
For example, 

EDITED_DESCRIPTIONS_CACHE_NAME = 'editedDescriptions'

Reply:
Checked and Fixed.